infrastructure-playbooks/lv-create.yml

```diff
@@ -0,0 +1,94 @@
+- name: creates logical volumes for the bucket index or fs journals on a single device and prepares for use of osd_scenario=lvm. 
+  hosts:
+  - osds
+
+  vars:
+    logfile: |
+      Suggested cut and paste into "group_vars/osds.yml" for configuring with osd_scenario=lvm
```

Comment:
Should we mention that these would get added to the ``lvm_volumes`` config option even if we don't actually print ``lvm_volumes`` here? I'm concerned that a user would copy and paste what's printed here directly into ``osds.yml`` and expect it to work.

Reply:
@andrewschoen updated. does it look fine now?